d8/d8batch/modules/d8feedsourcect/d8feedsourcect.module

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * Module file for the D8: Feed Source CT.
+ */
+use Drupal\Core\Entity\EntityStorageException;
+
+/**
+ * Implements callback_batch_operation().
```

Comment:
I don't think there's such a hook, so this should say `Batch operation callback for …`.